src/casp/services/deploy/cloudrun/default.json

```diff
@@ -3,22 +3,22 @@
     "cpu": "1000m",
     "memory": "512Mi",
     "max-instances": 100,
-    "min-instances": 0,
+    "min-instances": 1,
```

Comment:
Why set min-instances to 1?

Reply:
Ah, sorry, I set that for testing since it makes the APIs go a lot faster (e.g. I don't need to wait for the instance to spin up). I'll set back to 0 for now